api/src/main/java/com/flipkart/poseidon/validator/APIValidator.java

```diff
@@ -42,78 +42,81 @@
-                    System.exit(-1);
-                } catch (Exception e) {
-                    logger.error("Something went wrong while constructing CustomValidator", e);
-                    System.exit(-1);
                 }
-            }
 
-            if (args.length > 2) {
-                validateDataSources = true;
-                final String[] datasourcePackages = args[2].split(",");
-                packagesToScan = Arrays.asList(datasourcePackages);
-            }
-        }
-
-        List<String> validConfigs = new ArrayList<>();
-        try {
-            APIManager.scanAndAdd(dir, validConfigs);
-            for (String config : validConfigs) {
-                pojos.add(getMapper().readValue(config, EndpointPOJO.class));
+                if (args.length > 2) {
+                    validateDataSources = true;
+                    final String[] datasourcePackages = args[2].split(",");
+                    packagesToScan = Arrays.asList(datasourcePackages);
+                }
             }
 
-            Set<ClassPath.ClassInfo> classInfos = ClassPathHelper.getPackageClasses(Thread.currentThread().getContextClassLoader(), packagesToScan);
-            Map<String, Class<? extends DataSource<?>>> datasources = new HashMap<>();
-            for (ClassPath.ClassInfo classInfo : classInfos) {
-                Class clazz = Class.forName(classInfo.getName());
-                if (Modifier.isAbstract(clazz.getModifiers())) {
-                    continue;
+            List<String> validConfigs = new ArrayList<>();
+            try {
+                APIManager.scanAndAdd(dir, validConfigs);
+                for (String config : validConfigs) {
+                    pojos.add(getMapper().readValue(config, EndpointPOJO.class));
                 }
 
-                if (DataSource.class.isAssignableFrom(clazz)) {
-                    datasources.put(PoseidonLegoSet.getBlockId(clazz).get(), clazz);
+                Set<ClassPath.ClassInfo> classInfos = ClassPathHelper.getPackageClasses(Thread.currentThread().getContextClassLoader(), packagesToScan);
+                Map<String, Class<? extends DataSource<?>>> datasources = new HashMap<>();
+                for (ClassPath.ClassInfo classInfo : classInfos) {
+                    Class clazz = Class.forName(classInfo.getName());
+                    if (Modifier.isAbstract(clazz.getModifiers())) {
+                        continue;
+                    }
+
+                    if (DataSource.class.isAssignableFrom(clazz)) {
+                        datasources.put(PoseidonLegoSet.getBlockId(clazz).get(), clazz);
+                    }
                 }
-            }
 
-            Map<String, List<String>> errors = new HashMap<>();
-            for (EndpointPOJO pojo : pojos) {
-                List<String> pojoErrors = EndpointValidator.validate(pojo, datasources, validateDataSources);
+                Map<String, List<String>> errors = new HashMap<>();
```

Comment:
You seem to have different code style configured in your IDE. Please stick to Poseidon standards for Poseidon project to avoid these kind of space differences.